components/tools/OmeroPy/test/unit/clitest/test_user.py

```diff
@@ -23,7 +23,8 @@
 from omero.cli import CLI
 import pytest
 
-subcommands = ['add', 'list', 'password', 'email', 'joingroup', 'leavegroup']
+subcommands = ['add', 'list', 'password', 'email', 'joingroup', 'leavegroup',
```

Comment:
Wonder if this could be added to the "BaseControl" API? Somehow it should be possibly to figure out what commands are available...